contracts/test-helpers/ShareableMock.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity ^0.4.4;
+import "../Shareable.sol";
+
+contract ShareableMock is Shareable {
+
+  uint public count = 0;
+
+  function ShareableMock(address[] _owners, uint _required) Shareable(_owners, _required) {
+
+  }
+
+  function increaseCount(bytes32 action) onlymanyowners(action) {
+    count = count + 1;
+  }
+
+  function isOwnerConst(address _addr) constant returns (bool) {
+    return isOwner(_addr);
+  }
```

Comment:
What I mean is, make `isOwner` constant, remove `isOwnerConst` from `ShareableMock`, and use `isOwner` directly in tests. In the future, feel free to change the base contracts when you're writing tests if you find this problems.
We want the code to be as simple/correct as possible.